pkg/envoy/ads/stream.go

```diff
@@ -313,7 +313,7 @@ func isCNforProxy(proxy *envoy.Proxy, cn certificate.CommonName) bool {
 // is for the same service account as seen on the pod's service account
 func (s *Server) recordPodMetadata(p *envoy.Proxy) error {
 	if p.Kind() == envoy.KindGateway {
-		log.Debug().Msgf("Proxy with serial no %s is a gateway, skipping recording pod metadata", p.GetCertificateSerialNumber())
+		log.Debug().Msgf("Proxy with serial no %s is a Multicluster gateway, skipping recording pod metadata", p.GetCertificateSerialNumber())
```

Comment:
At some point, this was meant to be OSM gateway (ingress + multicluster). I'd assume the usage of a `gateway` in OSM to imply multicluster. In the future, if we decide to implement our gateway for whatever reason, another PR to s/multicluster// :-)

Reply:
For this sort of tagging, it's better to update the log line to something like:
`log.Debug().Str("context", "multicluster").Msgf("Proxy with serial no %s is a gateway, skipping recording pod metadata", p.GetCertificateSerialNumber())`.

Reply:
@shashankram That's a great idea - I'll create a task for us to do that for all log lines!!